Bayesian LMM Not all variables can be selected for univariate plots

boutinb opened the pull request:
Fixes jasp-stats/jasp-test-release#883
Add qsTr
Use currentValue in place of currentText when binding with dropdown
value
Put common QML code at one place

FBartos commented:
thanks Bruno!

TimKDJ commented on Resources/MixedModels/qml/MixedModelsLMM.qml:
```diff
@@ -535,13 +535,13 @@ Form {
-				{ label: "None",				value: "none"},
-				{ label: "Bonferroni",			value: "bonferroni"},
-				{ label: "Hommel",				value: "hommel"}
+				{ label: "Holm",					value: "holm"},
```
Wouldn't this also require `qsTr()` so it can be adapted in other writing systems?

FBartos commented on Resources/MixedModels/qml/MixedModelsBGLMM.qml:
```diff
@@ -230,397 +231,11 @@ Form {
-
-		Group
-		{
+	MM.MixedModelsBOption {}
```
an 's' is missing 
```suggestion
	MM.MixedModelsBOptions {}
```

boutinb committed:
Bayesian LMM Not all variables can be selected for univariate plots

Fixes jasp-stats/jasp-test-release#883
Add qsTr
Use currentValue in place of currentText when binding with dropdown
value
Put common QML code at one place

FBartos commented:
looks good to me
